common/lib/xmodule/xmodule/templates/problem/circuitschematic.yaml

```diff
@@ -3,9 +3,9 @@ metadata:
     display_name: Circuit Schematic Builder
     markdown: !!null
 data: |
-      <problem >
-        Please make a voltage divider that splits the provided voltage evenly.
-    
+      <problem>
+        <fieldset>
+          <legend>Make a voltage divider that splits the provided voltage evenly.</legend>
```

Comment:
Maybe use this text (adapted from the course authors guide):

Circuit schematic problems allow students to create virtual circuits by arranging elements such as voltage sources, capacitors, resistors, and MOSFETs on an interactive grid. The system evaluates a DC, AC, or transient analysis of the circuit.